.devcontainer/devcontainer.json

```
@@ -0,0 +1,65 @@
+{
+	"name": "Python 3",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "..",
+		"args": {
+			// Update 'VARIANT' to pick a Python version: 3, 3.10, 3.9, 3.8, 3.7, 3.6
+			// Append -bullseye or -buster to pin to an OS version.
+			// Use -bullseye variants on local on arm64/Apple Silicon.
+			"VARIANT": "3.10",
+			// Options
+			"NODE_VERSION": "lts/*"
+		}
+	},
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				"python.defaultInterpreterPath": "${workspaceFolder}/.venv/bin/python",
+				"python.formatting.blackPath": "${workspaceFolder}/.venv/bin/black",
+				"python.linting.enabled": true,
+				"python.linting.pylintEnabled": true,
+				"python.linting.flake8Path": "${workspaceFolder}/.venv/bin/flake8",
+				"python.linting.mypyPath": "${workspaceFolder}/.venv/bin/mypy",
+				"python.linting.pylintPath": "${workspaceFolder}/.venv/bin/pylint",
+				"python.testing.pytestArgs": [
+					"tests"
+				],
+				"python.testing.unittestEnabled": false,
+				"python.testing.pytestEnabled": true,
+				"python.venvPath": "${workspaceFolder}/.venv"
+			},
+
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"ms-python.python",
+				"ms-python.vscode-pylance",
+                "ms-azuretools.vscode-docker",
```

Absolute nitpick, feel free to ignore it

```suggestion
				"ms-azuretools.vscode-docker",
```